ADDING_AN_EXAMPLE.md

```diff
@@ -39,13 +39,19 @@ hope of keeping examples in sync with releases of dfx.
 
 # Documentation
 
-In repo dfinity/docs, add some general, language agnostic documentation for the
-example. E.g. for `hello`:
+Make sure to add documentation for the example in the [dfinity/portal](https://github.com/dfinity/portal) repository once your example has been merged.
```

Comment:
Are these changes on in this `ADDING_AN_EXAMPLE.md` file on purpose?

Reply:
yes, i branched off of https://github.com/dfinity/examples/tree/feature/transfer-from that's why the changes are included again here